### ISSUE TITLE
Docs about gnome-keyring assume Arch/Pacman, should have a Debian/Ubuntu reference also

### ISSUE THREAD

**Issue by aitor-gomila** [microsoft/vscode-docs#6114]
[This](https://code.visualstudio.com/docs/editor/settings-sync#_troubleshooting-keychain-issues) section of the docs speaking about GNOME Keyring should have a note for Debian/Ubuntu based systems. Most newbies use those systems, and without a proper note for those, they might get confused and frustrated.
![image](https://user-images.githubusercontent.com/46191980/223166426-eab92ab1-45f4-4f9b-9b74-e6022575d052.png)

**Comment by TylerLeonhardt** [comment]
Yes the only reason we assumed arch is because only a couple users hit this issue and they were all on arch. We didn't know that this issue existed on Debian/Ubuntu based systems.

When you modify that to work with `apt` (and I think maybe you need `gnome-keyring` instead of `libgnome-keyring` do the steps work?

**Comment by TylerLeonhardt** [comment]
For WSL, did you see this? https://stackoverflow.com/a/71879156

**Comment by aitor-gomila** [comment]
Apparently not

**Comment by TylerLeonhardt** [comment]
Does... it work? If so, we should add it to the docs.

**Comment by aitor-gomila** [comment]
No, not for me. Maybe I did it wrong. But it should probably be mentioned somewhere in the docs, I guess! :p

**Comment by TylerLeonhardt** [comment]
I'm gonna move this over to the main repo in case there's an obvious fix. I think this is just a matter of having a proper setup though.
